src/main/kotlin/com/autonomousapps/tasks/FindDeclarationsTask.kt

```diff
@@ -114,6 +114,8 @@ abstract class FindDeclarationsTask : DefaultTask() {
             )
           }
         }
+        .sortedWith(compareBy<Declaration> { it.configurationName }
```

Comment:
complementary with line 99 in this source file:
```
// We sort the map so that the task input property is consistent in different environments
return DeclarationContainer(mapping.toSortedMap())
```